<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleEmForm</class>
 <widget class="QWidget" name="SingleEmForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>95</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(101, 96, 218);</string>
  </property>
  <widget class="QLabel" name="dp_1_label">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>192</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>128</y>
     <width>62</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/switchstatus.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="em_label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>4</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/em_run.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vc_1_label">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>60</y>
     <width>13</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/vertical_connector_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="coro_text_1_label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>118</y>
     <width>24</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>正向</string>
   </property>
  </widget>
  <widget class="QLabel" name="inve_text_1_label">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>118</y>
     <width>24</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>反向</string>
   </property>
  </widget>
  <widget class="QLabel" name="coro_1_label">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>134</y>
     <width>17</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/coro.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inve_1_label">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>134</y>
     <width>17</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/inve.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
